src/App/styles.js

```diff
@@ -25,14 +25,21 @@ const useStyles = createUseStyles({
         }
     },
     body: {
-        textAlign: 'center',
-        backgroundColor: 'rgb(248, 98, 43)',
-        height: '100%',
         width: '75%',
+        padding: '5px',
```

Comment:
```suggestion
        padding: 5,
```
You can use just number for pixel sizes in JSS